app/src/main/java/com/poupa/vinylmusicplayer/dialogs/DeleteSongsDialog.java

```diff
@@ -92,7 +92,6 @@ private void deleteSongs(List<Song> songs, List<Uri> safUris) {
         MusicUtil.deleteTracks(getActivity(), songs, safUris, this::dismiss);
     }
 
-    @TargetApi(Build.VERSION_CODES.KITKAT)
```

Comment:
Not useful because kitkat is the youngest version we are targeting ?

Reply:
Exact.